testerum-backend/testerum-runner/testerum-runner-api/src/main/kotlin/com/testerum/runner/cmdline/report_type/builder/EventListenerProperties.kt

```diff
@@ -3,6 +3,16 @@ package com.testerum.runner.cmdline.report_type.builder
 object EventListenerProperties {
 
     object JsonEvents {
+        /**
+         * type     : String
+         * required : yes
+         *
+         * Required property that is an HTTP URL that points to the central reports server.
+         * If missing, an error is going to be triggered.
+         */
+//TODO Ionut: add a proper description. Specify what happens if the property is missing or if the reporting server is not found
+        const val REPORT_SERVER_URL = "reportServerURL"
```

Comment:
move this property to it's own object, since it's a different report; suggested name: ``RemoteServer``

Reply:
done